notebooks/Generating_Small_Debris_Population.ipynb

```diff
@@ -0,0 +1,475 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "b4384097",
+   "metadata": {},
+   "source": [
+    "This notebooks uses data from [ESA's MASTER tool](https://sdup.esoc.esa.int/) to create a population of small debris fragments between 1 and 10 cm."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "d7565d79",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "### Imports\n",
+    "%load_ext autoreload\n",
+    "%autoreload 2\n",
+    "\n",
+    "# Append main folder\n",
+    "import sys\n",
+    "sys.path.append(\"../\")\n",
+    "\n",
+    "import pykep as pk\n",
+    "import numpy as np\n",
+    "import matplotlib.pyplot as plt\n",
+    "import seaborn as sns\n",
+    "import pandas as pd\n",
+    "from tqdm import tqdm \n",
+    "from sklearn.neighbors import KernelDensity\n",
+    "\n",
+    "starting_t = pk.epoch_from_string('2022-01-01 00:00:00.000')\n",
+    "np.random.seed(42)\n",
+    "earth_radius = 6371\n",
+    "sats = pd.read_csv(\"../data/initial_population.csv\")\n",
+    "\n",
+    "# Utility functions\n",
+    "def get_shell_volume(lower_altitude,upper_altitude):\n",
+    "    # Subtraction of sphere volumes to get shell volume\n",
+    "    return 4/3 * np.pi * ((earth_radius+upper_altitude)**3-(earth_radius+lower_altitude)**3)\n",
+    "\n",
+    "def get_num_of_parts_in_shell(lower_altitude,upper_altitude,density):\n",
+    "    # Shell volume * density\n",
+    "    volume = get_shell_volume(lower_altitude,upper_altitude)\n",
+    "    nr = volume * density\n",
+    "#     print(volume,density,nr)\n",
+    "    return nr"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e4167304",
+   "metadata": {},
+   "source": [
+    "# Get estimated number of debris particles depending on altitude and size"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "aaf1df40",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Read in density data for 1-2.5cm, 2.5-5cm , ...\n",
+    "# Note it is expected that each file has densities for the same altitudes, this is not checked.\n",
+    "data_1cm = pd.read_csv(\"../data/MASTER_small_debris/1-2.5cm.csv\")\n",
+    "data_2cm = pd.read_csv(\"../data/MASTER_small_debris/2.5-5cm.csv\")\n",
+    "data_5cm = pd.read_csv(\"../data/MASTER_small_debris/5-7.5cm.csv\")\n",
+    "data_7cm = pd.read_csv(\"../data/MASTER_small_debris/7.5-10cm.csv\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "a1af951c",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "data_1cm"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "9a1447a2",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "num_debris_per_size = {} # will hold counts per altitude & size\n",
+    "sizes = [[1,2.5],[2.5,5.],[5,7.5],[7.5,10.0]]\n",
+    "datasets = [data_1cm,data_2cm,data_5cm,data_7cm]\n",
```

Comment:
why not directly initialize this as 
```python
datasets = [ pd.read_csv("...."), ... ]
```
Having the extra variables seems like a source for errors if we would change the bucketing at some point.